includes/classes/Database.class.php

```diff
@@ -186,6 +186,26 @@ public function selectSingle($qry, array $params = array(), $field = false)
 		$res = $stmt->fetch(PDO::FETCH_ASSOC);
 		return ($field === false || is_null($res)) ? $res : $res[$field];
 	}
+	
+	public function lists($table, $column, $key = null)
+	{
+		$selects = implode(', ', is_null($key) ? [$column] : [$column, $key]);
+		
+		$qry = "SELECT {$selects} FROM %%{$table}%%;";
+		$stmt = $this->_query($qry, [], 'select');
+
+		if (is_null($key)) {
+			while ($row = $stmt->fetch(PDO::FETCH_ASSOC)) {
+				$results[] = $row[$column];
+			}
+		} else {
+			while ($row = $stmt->fetch(PDO::FETCH_ASSOC)) {
+				$results[$row[$key]] = $row[$column];
+			}
+		}
+
+		return @$results;
```

Comment:
I think we should define, that the return value is an array and provide a empty array instad a null. Otherwise we had always to check its an array before using foreach.

An other fun fact: I have to the minium required php version to 5.3 to allow short syntax for array. *laugh*